docs/02.API-REFERENCE.md

```diff
@@ -1630,6 +1630,41 @@ jerry_value_is_undefined (const jerry_value_t value)
 
 - [jerry_release_value](#jerry_release_value)
 
+
+## jerry_value_strict_equal
```

Comment:
If we gave strict equal check api, then maybe we should also have the simple logic operators also (`<, >, ==`). However that would be another PR if we decide to implement it.